app/controllers/dashboards_controller.rb

```diff
@@ -0,0 +1,18 @@
+class DashboardsController < ApplicationController
+  before_action :authorize_supervisor
+
+  def show
+    payload = {
+      resource: { dashboard: ENV.fetch('METABASE_DASHBOARD_ID').to_i },
+      params: {},
+      exp: Time.now.to_i + (60 * 10) # 10 minute expiration
```

Comment:
why 10 mins expiry?

Reply:
It was the default set in the copy-pastable embed code. But I've checked and the expiration only needs to be as long as it takes the iframe to initially load, not to cover subsequent playing with widgets. So I've reduced to 10 seconds.

Reply:
I take it back - if you have the page open for a long time then play with the widget date selector it stops working after some built-in grace period. So I'll set this to an hour to make sure the data doesn't stop working mid-browse.